packages/tools/client-debugger/client-debugger/src/Registry.ts

```diff
@@ -122,6 +125,41 @@ export class DebuggerRegistry extends TypedEventEmitter<DebuggerRegistryEvents>
 			this.postRegistryChange();
 			return true;
 		},
+		["CONNECT_CONTAINER"]: (untypedMessage) => {
+			const message = untypedMessage as ConnectContainerMessage;
+			const { containerId } = message.data;
+			const existingDebugger = this.registeredDebuggers.get(containerId);
+			if (existingDebugger === undefined) {
+				console.warn(`No active debugger found for container ID "${containerId}".`);
+				return false;
+			}
+			existingDebugger.container?.connect();
```

Comment:
For the three handlers, let's do an explicit check for `container` being undefined. If we encounter that, we should `console.warn()` about it, it's probably unexpected.

Reply:
I don't think the `container` property can be undefined on the debugger. I'm not sure the `?` is necessary.

Reply:
Updated.